utilities/transactions/transaction_lock_mgr.h

```diff
@@ -27,7 +27,7 @@ struct LockMap;
 struct LockMapStripe;
 
 class Slice;
-class TransactionDBImpl;
+class PessimisticTxnDB;
```

Comment:
ah it became difficult. OptimisticTransactionDB is in the include so we cannot rename it.
How about renaming PessimisticTxn to PessimisticTransaction but keep the short Txn for its children?